qprofiler2/test/org/qcmg/qprofiler2/OptionsTest.java

```diff
@@ -4,32 +4,32 @@
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 import org.junit.Test;
-import org.qcmg.qprofiler2.Options2;
+import org.qcmg.qprofiler2.Options;
 
-public class OptionsTest {
+public class OptionsTest { 
```

Comment:
In several places in this file there is now a new trailing space after open `{` blocks that shouldn't be there.